code/src/components/wrapper.js

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from "react";
+
+import Thoughts from "./thoughts";
+import Form from "./form";
+import Loading from "./loader";
+
+import { API_URL, LIKES_URL } from "utils/urls";
+
+const Wrapper = () => {
+const [thoughts, setThoughts] = useState([]);
+const [newThought, setNewThought] = useState("");
+const [loading, setLoading] = useState(false);
+
+useEffect(() => {
+    fetchThoughts();
+}, []);
```

Comment:
This was really nice! When I watched the lecture when this was explained, I got so confused... I decided to skip this part in my project, but when I see it your project now, it is so clear! well done!